### PR TITLE
[arrow]replace Abi.h with arrow's abi.h

### PR DESCRIPTION
**Pull request by zhejiangxiaomai**
Remove unuseful file Abi.h.

### CODE REVIEW

File: velox/vector/arrow/CMakeLists.txt [hunk 1]
```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 add_library(velox_arrow_bridge Bridge.cpp)
 
-target_link_libraries(velox_arrow_bridge velox_memory velox_type velox_buffer
-                      velox_exception)
+target_link_libraries(velox_arrow_bridge arrow velox_memory velox_type
```
Comment (Yuhta) [inline velox/vector/arrow/CMakeLists.txt:16]: We don't want to depend on arrow in VELOX_MINIMAL here

**Comment by Yuhta** [comment]
What is the reason behind this change?  We are keeping a copy of `Abi.h` because we don't want to restrict arrow dependency to tests and parquet for now.  It's a dependency too large and can cause trouble if we use it everywhere.

**Comment by zhejiangxiaomai** [comment]
> We are keeping a copy of `Abi.h` because we want to restrict arrow dependency to tests and parquet only for now. It's a dependency too large and can cause trouble if we use it everywhere like this.



> We are keeping a copy of `Abi.h` because we want to restrict arrow dependency to tests and parquet only for now. It's a dependency too large and can cause trouble if we use it everywhere like this.

Ok,  I know.

**Comment by Yuhta** [comment]
Thanks for the understanding

**Pull request closed without merging**